src/main/java/com/dev/cinema/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -83,23 +84,33 @@ public void update(MovieSession movieSession) {
     }
 
     @Override
-    public void remove(MovieSession movieSession) {
+    public void remove(Long id) {
         Transaction transaction = null;
         Session session = null;
         try {
             session = sessionFactory.openSession();
             transaction = session.beginTransaction();
+            MovieSession movieSession = session.get(MovieSession.class, id);
```

Comment:
we can even assume that the movieSession with this id exists and use `session.load()` instead of `session.get()` - what do you think?

Reply:
sure, i used the method load() instead get()